src/fabric/src/fabric_db_delete.erl

```diff
@@ -81,10 +81,10 @@ maybe_stop(W, Counters) ->
             {#shard{dbname=Name}, _} = hd(Counters),
             couch_log:warning("~p not_found ~s", [?MODULE, Name]),
             {stop, not_found};
-        {W, _, _} ->
-            {stop, ok};
         {N, M, _} when N >= (W div 2 + 1), M > 0 ->
-            {stop, accepted};
```

Comment:
@kocolosk as the original decision maker for these semantics, do you have a problem with this? We've accepted the same idea on the create db side but before we accept this one, and make a formal release including both changes, I wanted to hear your thoughts. The idea here is that you should not get a 500 error when creating or deleting a db in a degraded cluster.